<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-J41V6O34" ignored="1" name="Car loan" tests="10" failures="1" timestamp="03 四月 2019 22:58:58 GMT" time="0.148" errors="0">
  <testcase name="MobileTest" time="0.01" classname="test.Class3"/>
  <testcase name="MobileTest2" time="0.011" classname="test.Class3">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at test.Class3.MobileTest2(Class3.java:32)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- MobileTest2 -->
  <testcase name="MobileTest3" time="0.004" classname="test.Class3"/>
  <testcase name="MobileTest4" time="0.002" classname="test.Class3"/>
  <testcase name="Weblogin" time="0.001" classname="test.Class3"/>
  <testcase name="MobileMenu1Test" time="0.009" classname="test.Class4"/>
  <testcase name="MobileMenu2Test" time="0.002" classname="test.Class4"/>
  <testcase name="TimeOutTest" time="0.013" classname="test.Class4"/>
  <testcase name="WebCostcologdin" time="0.002" classname="test.Class4"/>
  <testcase name="MobileAndroidTest" time="0.002" classname="test.Class4"/>
  <testcase name="MobileAppiumTest" time="0.0" classname="test.Class4">
    <ignored/>
  </testcase> <!-- MobileAppiumTest -->
</testsuite> <!-- Car loan -->
